examples/jenkins/jenkins-persistent-template.json

```diff
@@ -204,11 +195,35 @@
       "value": "password"
     },
     {
+      "name": "MEMORY_REQUEST",
+      "displayName": "Memory Lower Limit",
+      "description": "Reserved amount of memory for the container.",
+      "value": "1200Mi"
+    },
+    {
+      "name": "MEMORY_LIMIT",
+      "displayName": "Memory Upper Limit",
+      "description": "Maximum amount of memory the container can use.",
+      "value": "2000Mi"
+    },
```

Comment:
> you are quota'd against your upper limit.
ok let's keep it at 500Mi

> Would like to know why you're using the dockerhub jenkins image instead of our image?

Our image has old version of Jenkins, that's the only reason. But my point with this template was just to get specification of different images streams working.